src/ops/noop.cc

```diff
@@ -118,44 +120,38 @@ void NoOp::init(const FFModel& ff)
     Runtime* runtime = ff.config.lg_hlr;
     set_argumentmap_for_init(ff, argmap);
     IndexLauncher launcher(NOOP_INIT_TASK_ID, parallel_is,
-                           TaskArgument(NULL, 0), argmap,
-                           Predicate::TRUE_PRED, false/*must*/, 0/*mapper_id*/,
+                           TaskArgument(NULL, 0), argmap, Predicate::TRUE_PRED,
+                           false /*must*/, 0 /*mapper_id*/,
                            outputs[0]->machine_view.hash());
     FutureMap fm = runtime->execute_index_space(ctx, launcher);
     fm.wait_all_results();
     set_opmeta_from_futuremap(ff, fm);
   }
 }
 
-void NoOp::forward(const FFModel& ff)
-{}
+void NoOp::forward(const FFModel& ff) {}
 
-void NoOp::backward(const FFModel& ff)
-{}
+void NoOp::backward(const FFModel& ff) {}
 
-bool NoOp::measure_operator_cost(
-    Simulator* sim,
-    const MachineView& mv,
-    CostMetrics& cost_metrics) const {
+bool NoOp::measure_operator_cost(Simulator* sim, const MachineView& mv,
+                                 CostMetrics& cost_metrics) const {
   cost_metrics.forward_time = 0.0f;
   cost_metrics.backward_time = 0.0f;
   cost_metrics.memory_requirement = 0;
   return true;
 }
 
-size_t NoOp::get_params_hash() const {
-  size_t hash = 0;
-  for (int i = 0; i < this->numInputs; i++) {
-    hash_combine(hash, this->inputs[i]);
+// TODO: optimize the performance
```

Comment:
TODO probably unnecessary, I highly doubt this method will ever be a bottleneck. What performance characteristics are you worried about?